dashboard/modules/serve/serve_head.py

```diff
@@ -58,9 +53,9 @@ async def delete_serve_application(self, req: Request) -> Response:
     @routes.put("/api/serve/deployments/")
     @optional_utils.init_ray_and_catch_exceptions(connect_to_serve=True)
     async def put_all_deployments(self, req: Request) -> Response:
-        serve_application_json = await req.json()
+        serve_application_text = await req.text()
         serve_application_schema = ServeApplicationSchema.parse_raw(
```

Comment:
huh, why doesn't this take json?

Reply:
It does take json, but I left it in text form there. `ServeApplicationSchema.parse_raw()` should convert the text into the schema.